acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/web3j/privacy/OnChainPrivacyAcceptanceTest.java

```diff
@@ -103,6 +106,96 @@ public void canAddParticipantToGroup() {
     charlie.verify(privateTransactionVerifier.existingPrivateTransactionReceipt(commitmentHash));
   }
 
+  @Test
+  public void aliceCanAddRemoveAddBob() {
+    final String privacyGroupId = createOnChainPrivacyGroup(alice);
+
+    checkOnChainPrivacyGroupExists(privacyGroupId, alice);
+
+    alice.execute(privacyTransactions.privxLockPrivacyGroupAndCheck(privacyGroupId, alice));
+    alice.execute(privacyTransactions.addToPrivacyGroup(privacyGroupId, alice, bob));
```

Comment:
Same here, it would be good to use the "concise" version:
```
addMembersToPrivacyGroup(privacyGroupId, alice, bob);
```

Reply:
Done